hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/ozone/container/common/TestEndPoint.java

```diff
@@ -165,17 +165,95 @@ public void testGetVersionTask() throws Exception {
     }
   }
 
+
+  /**
+   * Writing data to tmp dir and checking if it has been cleaned.
+   */
   @Test
-  public void testCheckVersionResponse() throws Exception {
+  public void testTmpDirCleanup() throws Exception {
     OzoneConfiguration conf = SCMTestUtils.getConf();
     conf.setBoolean(OzoneConfigKeys.DFS_CONTAINER_IPC_RANDOM_PORT,
         true);
     conf.setBoolean(OzoneConfigKeys.DFS_CONTAINER_RATIS_IPC_RANDOM_PORT,
         true);
-    conf.setBoolean(OzoneConfigKeys.DFS_CONTAINER_RATIS_DATASTREAM_ENABLED,
```

Comment:
Looks like this test had ratis streaming enabled but it got turned off by accident. May have happened when resolving conflicts from master.

Reply:
I see a similar thing in `testGetVersionTask`

Reply:
Yes, that must have been an accident. I only meant to cleanup the `setUp()`.